### ISSUE TITLE
Milton 2.7.1.3: Unresolved dependencies: Milton repo unreachable

### ISSUE THREAD

**Issue by omulmann** [miltonio/milton2#71]
**Problem**
Milton 2.7.1.3 built failed

**Cause**

Invalid repo:

``` XML
        <repository>
            <id>milton-repo</id>
            <url>http://milton.io/maven/</url>
        </repository>
```

and default Maven repo

**Impact**

Unresolved dependencies

E.g. for `milton-client`, the deps below cannot be resolved:

http://milton.io/maven/io/milton/milton-api/2.7.1.3/milton-api-2.7.1.3.pom
https://repo.maven.apache.org/maven2/io/milton/milton-api/2.7.1.3/milton-api-2.7.1.3.pom
http://milton.io/maven/io/milton/cardme/1.0.0/cardme-1.0.0.pom
https://repo.maven.apache.org/maven2/io/milton/cardme/1.0.0/cardme-1.0.0.pom

**Comment by damnhandy** [comment]
This is related to issues #75 and #77

**Comment by damnhandy** [comment]
Seems to be resolved now, checkout issue #77

**Comment by bradmac** [comment]
Moved repos to bintray
